packages/web/src/pages/chat-page/components/ChatBlastAudienceDisplay.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  useAudienceUsers,
+  useChatBlastAudienceContent
+} from '@audius/common/hooks'
+import { User } from '@audius/common/models'
+import {
+  Text,
+  IconTowerBroadcast,
+  Paper,
+  Flex,
+  IconInfo
+} from '@audius/harmony'
+import { ChatBlast } from '@audius/sdk'
+
+import { UserProfilePictureList } from 'components/notification/Notification/components/UserProfilePictureList'
+import { Tooltip } from 'components/tooltip'
+
+const USER_LIST_LIMIT = 10
+
+const messages = {
+  title: 'Send a Message Blast',
+  description: 'Send direct messages in bulk.',
+  tooltip: 'Replies will appear in your inbox as new threads.'
+}
+
+interface ChatBlastAudienceDisplayProps {
+  chat: ChatBlast
+}
+
+export const ChatBlastAudienceDisplay = (
+  props: ChatBlastAudienceDisplayProps
+) => {
+  const { chat } = props
+
+  const { audienceCount } = useChatBlastAudienceContent({ chat })
+
+  const users = useAudienceUsers(chat, USER_LIST_LIMIT + 1)
```

Comment:
why the +1?

Reply:
ah because we want one extra picture for the background of the overflow tile in the user list.
The one that says +X still needs a user behind it, so if we display 10 real ones and an overflow, we need 11 real pictures

Reply:
I'll add a comment